src/main/java/org/checkerframework/checker/androidresource/AndroidResourceAnnotatedTypeFactory.java

```diff
@@ -219,11 +228,52 @@
     protected final AnnotationMirror XML_CONTAINER =
             AnnotationBuilder.fromClass(elements, XmlContainer.class);
 
+    /**
+     * Aliases for Android @{@code XXXRes} annotations.

```

Comment:
These are not aliases and not annotations. This field contains package prefixes for the actual annotations.
Please pick a more suitable name for the field and describe it correctly.

Reply:
Fixed.